projects/packages/forms/src/blocks/field-telephone/index.js

```diff
@@ -0,0 +1,48 @@
+import { Icon } from '@wordpress/components';
```

Comment:
Sometimes I in the jetpack code base I see 
```
import { Icon } from '@wordpress/components';
import { mobile } from '@wordpress/icons';
```
and other times
I see 
```
import { Icon, mobile } from '@wordpress/icons'; 
```
I am assuming the end result is the same thing. Does it make sense to just use the @wordpress/icons package ?

Since that Icon component is a bit more light wight?

Reply:
Both Icon components are light weight, so I don't think that aspect makes much difference in the scheme of things.

For me, I'd lean towards using `Icon` from the `@wordpress/components` package:
- This package is effectively a source of truth for WordPress components
- Contains some size logic to help ensure icons align with WordPress design systems
- There's also some validation before attempting to pass through additional props
- More flexible for future updates
- Jetpack already has `@wordpress/components` as a dependency